### ISSUE TITLE
i just duped an item in the hand tele

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#33700]
(WEB REPORT BY: windowserrors REMOTE: 206.221.180.138:7777)
# Revision
4d311016e03ce03de2499ef2ed2fda8317224685

# Description

i duped my ID

# Steps to Reproduce

i was using a hand tele to solo the rainbow alert in the captains office and the hop office and after i set rainbow alert i was standing on top of the portal and *putting my ID back inside my PDA* but for some reason it didn't go in so i stepped off the portal and went back to the hop office and there was a second ID exactly the same as the one in my PDA

**Comment by Eneocho** [comment]
Yeah, using stuff while on top of a portal accidentally teleports it. This probably isn't the only the dupe exploit with portals.

**Comment by Exxion** [comment]
There (probably) aren't any dupe exploits with portals. The ID on the floor and the one in your PDA are one and the same. Removing the ID from your PDA will teleport the floor one to your hand.
Still an exploit, just not a dupe one. The way BYOND handles storing objects makes it very easy to avoid dupe exploits. One of its few good features.

**Comment by Eneocho** [comment]
mildly exploitable teleport trick you could use with a friend
